### DISCUSSION TITLE
Multiplayer lobby hotkeys

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#20520] [Q&A]
I want to add hotkeys for ready/spectate buttons.

- Where this should be implemented - inside buttons via click triggering, or on lobby screen level?
- Which keys should be by default? Enter is already used by chat.

**Comment by peppy** [comment]
Hard to say really. I'd hope that if implementing any shortcuts, they would use existing global bindings where appropriate.

Enter being used by chat can potentially be avoided as an issue by allowing chat to not handle enter when there's no text.
